matrix/tasks/reset.py

```diff
@@ -0,0 +1,12 @@
+import asyncio
+
+
+async def reset(context, rule, task, event=None):
```

Comment:
Yeah, I'm not sure we should be handing matrix (or BT) a model that they can work within, or a controller with permissions such that it can create new models, but not see or affect any models that it didn't create itself.  I'm not sure we even have granular enough permissions to do the latter yet.